secrets/src/traits.rs

```diff
@@ -0,0 +1,133 @@
+pub trait Classify {
```

Comment:
I think ideally the traits in this file could use some comments.

Reply:
Added some.